test/unit/v1.0.0/converter-v1-to-v2.test.js

```diff
@@ -493,14 +485,135 @@ describe('v1.0.0 to v2.0.0', function () {
                             schema: 'https://schema.getpostman.com/json/collection/v2.0.0/collection.json'
                         },
                         item: [{
-                            auth: null,
                             item: []
                         }]
                     });
                     done();
                 });
             });
         });
+
+        describe('prioritizeV2: true', function () {
```

Comment:
Can you also add tests for

1. missing legacy
2. missing non-legacy properties.
2. both legacy and non-legacy auth missing

tests for `prioritizeV2` for events and `preRequestScript` and `tests`

Reply:
@kunagpal can we add the same tests for `prioritize:false`